python/dask_cudf/dask_cudf/io/csv.py

```diff
@@ -56,6 +56,11 @@ def read_csv(path, blocksize="default", **kwargs):
         Passthrough key-word arguments that are sent to
         :func:`cudf:cudf.read_csv`.
 
+    Notes
+    -----
+    When either of `skipfooter`/`skiprows`/`nrows` are
+    `blocksize` will default to None if nothing is passed.
```

Comment:
Something's not right here :D
```suggestion
    When either of `skipfooter`/`skiprows`/`nrows` are passed
    `blocksize` will default to None.
```
I'm not 100% sure on the grammar, the previous version seems to have been borked in editing.